pkg/certs/cert-inspection/certgraphanalysis/analyzer.go

```diff
@@ -77,6 +85,92 @@ func InspectConfigMap(obj *corev1.ConfigMap) (*certgraphapi.CertificateAuthority
 		return nil, err
 	}
 	caBundleDetail = addConfigMapLocation(caBundleDetail, obj.Namespace, obj.Name)
+	return caBundleDetail, nil
+}
+
+func extractKubeConfigFromConfigMap(obj *corev1.ConfigMap) (*rest.Config, error) {
```

Comment:
might be more useful to return the actual kubeconfig type since it can contain multiple stanzas and rest.Config only holds one.

Reply:
Good idea, I'll implement that

Reply:
This would require reworking entire `InspectSecret/InspectConfigMap` as they are expected to return just one secret. I'll implement this in a separate PR

Reply:
Followup - https://github.com/openshift/library-go/pull/1751